vernac/declaremods.ml

```diff
@@ -1584,10 +1588,10 @@ let end_library ~output_native_objects dir =
 
 (** {6 Iterators} *)
 
-let iter_all_segments f =
+let iter_all_interp_segments f =
   let rec apply_obj prefix obj = match obj with
     | IncludeObject aobjs ->
-      let objs = expand_aobjs aobjs in
+      let objs = InterpVisitor.expand_aobjs aobjs in
```

Comment:
Originally I also did the synterp objects but search output got lots of duplicates for some reason so now it's just interp

Reply:
It's only used by Search so whatever you put in there must follow the old behaviour.